orchestration/src/main/resources/spec/orchestration.yaml

```diff
@@ -1,103 +1,89 @@
 openapi: 3.0.0
+
 info:
-  title: Orchestration
-  description: Orchestration is an inference service which provides common additional capabilities for business AI scenarios, such as content filtering and data masking. At the core of the service is the LLM module which allows for an easy, harmonized access to the language models of gen AI hub. The service is designed to be modular and extensible, allowing for the addition of new modules in the future. Each module can be configured independently and at runtime, allowing for a high degree of flexibility in the orchestration of AI services.
-  version: 0.29.3
-x-sap-shortText: Enhance content generation with additional capabilities for business AI scenarios.
+  title: Internal Orchestration Service API
+  description: SAP AI Core - Orchestration Service API
+  contact:
+    name: SAP AI Core
+  version: 0.0.1
```

Comment:
The spec is the latest commit on the main branch of the orchestration repository

Reply:
I can't accept this drive-by change.

Reply:
I reverted to 0.36.1 (it still says 0.0.1)

Reply:
Why is this not [0.29.3](https://api.sap.com/api/ORCHESTRATION_API/overview) ?